### ISSUE TITLE
backup of encrypted data

### ISSUE THREAD

**Issue by OmlineEditor** [hardcore-sushi/DroidFS#67]
now you can export files in an unencrypted type from a hidden volume. it's not safe. I ask you to make a backup function when all the data is packed in a zip archive in encrypted form.

that is, it is a simple data packaging in a zip archive with the name:  _containerName_date_time.zip_

you can also implement data recovery from a backup from a zip archive.

**Comment by hardcore-sushi** [comment]
I don't know if we really need this, as we can just create an encrypted volume on the shared storage and import all files from the hidden volume there. It's safe and doesn't require additional code into the app.

**Comment by OmlineEditor** [comment]
how do I backup data? creating one zip file is the easiest way, and this zip file can also be sent by mail or uploaded to cloud storage

**Comment by hardcore-sushi** [comment]
If you really want a zip file, just zip the encrypted shared volume.

**Comment by OmlineEditor** [comment]
I can't encrypt a hidden volume. I couldn't get into the folder where it is located.
that's why I'm asking you to implement this function.
can you do it?

**Comment by hardcore-sushi** [comment]
Yes but you can import all of the hidden volume content into a non-hidden volume and zip this one. I could implement a backup feature but that's not the priority since we have this workaround.

**Comment by OmlineEditor** [comment]
an easier way to get a backup is welcome and desirable. Taking care of users is the main thing. make a convenient way for users to get a backup.
